### PR TITLE
Add porting config for Cockroach-db

### PR DESCRIPTION
**Pull request by KSTARK007**

### CODE REVIEW

File: data/cockroach_cockroachDB/config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "deploy": {
+        "steps": [
+            {
+                "apply": {
+                    "file": "data/kiran/operator.yaml",
```
Comment (tylergu) [inline data/cockroach_cockroachDB/config.json:6]: @KSTARK007 Could you fix the path here?

Comment (KSTARK007) [reply data/cockroach_cockroachDB/config.json]: yes fixed the path. Please verify

**Comment by KSTARK007** [comment]
Fixed the path error

**Comment by KSTARK007** [comment]
I update the path for seed CR too. I am soo sorry I missed it the first time around

**Pull request merged**